test/integration/test_serialization.py

```diff
@@ -25,13 +26,14 @@ def create_key_pair_and_serialize(tmp_location: Path, q: mp.Queue, default_asset
         raise e
 
 
-def test_keypair_manager_with_local_stack(tmp_path, local_stack, default_asset_id):
+def test_keypair_manager_with_local_stack(tmp_path, local_stack_aws_access, default_asset_id):
     """
-    Test that serialization and deserialization of KeyFileManager work!
+    Test that serialization and deserialization of KeyFileManager works
     """
     tmp_file = Path(tmp_path) / "key_file_manager.data"
     q = mp.Queue()
-    p = mp.Process(target=create_key_pair_and_serialize, args=(tmp_file, q, default_asset_id))
+    p = mp.Process(target=create_key_pair_and_serialize, args=(
+        local_stack_aws_access, tmp_file, q, default_asset_id))
```

Comment:
Forwarding local_stack_aws_access to the process is, in my option, not the best idea, because it might get serialized.